tracing/src/main/java/org/axonframework/extensions/tracing/TraceableSubscriptionQueryResult.java

```diff
@@ -0,0 +1,55 @@
+package org.axonframework.extensions.tracing;
+
+import io.opentracing.Span;
+import org.axonframework.common.Registration;
+import org.axonframework.queryhandling.SubscriptionQueryResult;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/**
+ * Traceable implementation of {@link SubscriptionQueryResult}.
+ *
+ * @param <I> The type of initial result
+ * @param <U> The type of incremental updates
+ * @author Lucas Campos
+ * @since 4.2
```

Comment:
Think we're gonna push this to 4.3, so the since tag should be adjusted accordingly. :-)

Reply:
👍